resources/public/api/conf/2.0/schemas/retrieveBalanceAndTransactions/retrieve_self_assessment_balanceDetails_response.json

```diff
@@ -0,0 +1,119 @@
+              }
+            }
+          ]
+        },
+        "earliestPaymentDateOverdue": {
+          "description": "The next day after the payment due date in the format YYYY-MM-DD",
+          "type": "string",
+          "example": "2018-04-05"
+        },
+        "totalBalance": {
+          "description": "The sum of all the unpaid charges excluding statistical and  EOY coded out. The total balance should be equal to the sum of the balance due within 30 days, balance not due in 30 days, and overdue amount.\r\nThe value must be between -99999999999.99 and 99999999999.99 to 2 decimal places.",
+          "type": "number",
+          "example": 5009.99
+        },
+        "amountCodedOut": {
+          "description": "The total amount of underpayment codeout.\r\nThe value must be between -99999999999.99 and 99999999999.99 to 2 decimal places.",
+          "type": "number",
+          "example": 5009.99
+        },
+        "totalBcdBalance": {
+          "description": "The total of the Balance charge debit balance amount.\r\nThe value must be between -99999999999.99 and 99999999999.99 to 2 decimal places",
+          "type": "number",
+          "example": 5009.99
+        },
+        "unallocatedCredit": {
+          "description": "Sum of all the Open credit Items on the customer's account, posted for Income tax Self-assessment.\r\nThe value must be between -99999999999.99 and 99999999999.99 to 2 decimal places.",
+          "type": "number",
+          "example": 5009.99
+        },
+        "allocatedCredit": {
+          "description": "Sum of all the closed credit items on the customer's account, posted for  Income tax Self-assessment.\r\nThe value must be between -99999999999.99 and 99999999999.99 to 2 decimal places.",
+          "type": "number",
+          "example": 5009.99
+        },
+        "totalCredit": {
+          "description": "Sum of unallocated credit + allocated credit.\r\nThe value must be between -99999999999.99 and 99999999999.99 to 2 decimal places.",
+          "type": "number",
+          "example": 5009.99
+        },
+        "firstPendingAmountRequested": {
+          "description": "This will be the first pending amount that is requested.\r\nThe value must be between -99999999999.99 and 99999999999.99 to 2 decimal places.",
+          "type": "number",
+          "example": 5009.99
+        },
+        "secondPendingAmountRequested": {
+          "description": "This will be the second pending amount that is requested.\r\nThe value must be between -99999999999.99 and 99999999999.99 to 2 decimal places.",
+          "type": "number",
+          "example": 5009.99
+        },
+        "availableCredit": {
+          "description": "This will be equal to total credit minus the pending amount(s) requested.\r\nThe value must be between -99999999999.99 and 99999999999.99 to 2 decimal places.",
+          "type": "number",
+          "example": 5009.99
+        }
+      },
+      "required": [
```

Comment:
only "payableAmount", "pendingChargeDueAmount", "overdueAmount", "totalBalance" should be required